fields/types/localfiles/LocalFilesType.js

```diff
@@ -170,7 +171,7 @@ localfiles.prototype.addToSchema = function() {
 		delete: function(element_id) {
 			if (exists(this, element_id)) {
 				var values = this.get(field.path);
-				var value = values.find(function (val) { return val._id === element_id; });
+				var value = values.find(function (val) { return val._id == element_id; });
```

Comment:
If what we want to accomplish is to compare values with type coercion I think `==` is better suited than `String(x) === String(y)` since that's exactly what the `==` operator is for :) Maybe with a comment as you said, just because one might think that `val._id` is a `String`.